code/controllers/configuration.dm

```diff
@@ -13,6 +13,9 @@
 /datum/protected_configuration/vv_get_var(var_name)
 	return debug_variable(var_name, "SECRET", 0, src)
 
+/datum/protected_configuratoin/SDQL_update()

```

Comment:
So.. doc, what's the deal with my leg muscle? Why isn't it moving?

It was toin.